homework-3/generate_phrase.py

```diff
@@ -0,0 +1,43 @@
+"""
+Create required phrase.
+----------------------
+
+You are given a string of available characters and a string representing a word or a phrase that you need to generate.
+Write a function that checks if you can generate required word/phrase using the characters provided.
+If you can, then please return True, otherwise return False.
+
+NOTES:
+    You can only generate the phrase if the frequency of unique characters in the characters string is equal or greater
+    than frequency in the document string.
+
+FOR EXAMPLE:
+
+    characters = "cbacba"
+    phrase = "aabbccc"
+
+    In this case you CANNOT create required phrase, because you are 1 character short!
+
+IMPORTANT:
+    The phrase you need to create can contain any characters including special characters, capital letter, numbers
+    and spaces.
+
+    You can always generate an empty string.
+
+"""
+
+from collections import Counter
+
+def generate_phrase(characters, phrase):
+    new_characters = characters.lower()
+    new_phrase = phrase.lower()
+    characters_counts = Counter(new_characters)
+    phrase_counts = Counter(new_phrase)
+    return all(characters_counts.get(characters, 0) >= count for characters, count in phrase_counts.items())
```

Comment:
Hi Lujein, this looks really good. However, I am sorry I would have to flag this as plagiarism because looks like an exact lift from https://stackoverflow.com/a/73198472. From that link, you will find the exact extract from a portion of the function created there that match exactly what you have implemented.
This means that I would have to score this homework zero.

At CFG, we have zero tolerance for plagiarism. However, please do reach out if you believe this is a misunderstanding.